docs/architecture/adr-014-proportional-slashing.md

```diff
@@ -84,7 +76,3 @@ Proposed
 - Increases decentralization by disincentivizing delegating to large validators
 - Incentivizes Decorrelation of Validators
 - More severely punishes attacks than accidental faults
-
-### Negative
```

Comment:
Are there any Neutral consequences?

Reply:
Idk, none that I can think of.  What would be a neutral consequence?